test/functional/deprecate_warning_tests.js

```diff
@@ -23,8 +24,9 @@ describe('Deprecation Warnings', function() {
   it('node --no-deprecation flag should suppress all deprecation warnings', {
     metadata: { requires: { node: '>=6.0.0' } },
     test: function(done) {
+      const fileLocation = path.join(process.cwd(), '../test/tools/deprecate_warning_test_program.js');
```

Comment:
Maybe we should just move `test/tools/deprecate_warning_test_program.js` to the right spot instead of modifying this file.

Reply:
`deprecate_warning_test_program.js` is already in the `test/tools` folder as specified in the original file. However, the program seems to execute from the `test-runner-poc` folder. I'm not sure if there is any way to move the `deprecate_warning_test_program.js` file to the right place unless I duplicate the `test/tools/` folder inside the `test-runner-poc` folder

Reply:
I think that would be preferable to changing the test itself. Let's try to minimize changes to tests in this PR.

Reply:
Ok! moved the `deprecate_warning_test_program.js` into `test-runner-poc/test/tools`

Reply:
And undid the changes to the file paths in the test file